src/components/policy/policy_regular/test/policy_manager_impl_test.cc

```diff
@@ -834,6 +834,35 @@ TEST_F(
   EXPECT_EQ("UP_TO_DATE", manager->GetPolicyTableStatus());
 }
 
+TEST_F(PolicyManagerImplTest2, UpdateApplication_AppServices) {
+  // Arrange
+  CreateLocalPT("sdl_preloaded_pt.json");
+  EXPECT_EQ("UP_TO_DATE", manager->GetPolicyTableStatus());
+  GetPTU("valid_sdl_pt_update.json");
+  EXPECT_EQ("UP_TO_DATE", manager->GetPolicyTableStatus());
+  // Try to add existing app
+  policy_table::AppServiceParameters app_service_parameters =
+      policy_table::AppServiceParameters();
+  manager->GetAppServiceParameters(app_id2, &app_service_parameters);
+
+  ASSERT_FALSE(app_service_parameters.find("MEDIA") ==
+               app_service_parameters.end());
+
+  auto service_names = *(app_service_parameters["MEDIA"].service_names);
+
+  EXPECT_TRUE(service_names.is_initialized());
+  EXPECT_EQ(static_cast<std::string>(service_names[0]), "SDL App");
+  EXPECT_EQ(static_cast<std::string>(service_names[1]), "SDL Music");
```

Comment:
Might need a length check as well, don't know what will happen when you go out of bounds.